lib/chef/knife/bootstrap.rb

```diff
@@ -30,6 +30,7 @@ class Bootstrap < Knife
 
       SUPPORTED_CONNECTION_PROTOCOLS ||= %w{ssh winrm}.freeze
       WINRM_AUTH_PROTOCOL_LIST ||= %w{plaintext kerberos ssl negotiate}.freeze
+      CHEF_15 ||= 15
```

Comment:
This constant seems a bit pointless. Just put 15 in the comparison

Reply:
@tas50 I think you are reviewing the wrong PR.
Could you please have a look at https://github.com/chef/chef/pull/8782.

Going to close this PR because it's against 15